src/actions/notificationsActions.js

```diff
@@ -117,7 +118,7 @@ const onFirebaseMessageAction = (message: FirebaseMessage) => {
   };
 };
 
-const hasFCMPermission = async () => {
+export const hasFCMPermission = async () => {
   const status = await firebaseMessaging.requestPermission();
```

Comment:
This may fail dependant on the security settings on the device of the user. Please add a `.catch` condition here.

Reply:
We implemented the .catch condition here.